dist/READMEcontainer.md

```diff
@@ -0,0 +1,70 @@
+ovn container construction and operation
+
+NOTES/LIMITS:
+- This only works on a cluster with a single master. Openshift
+  supports multiple masters.
+- Single network cidr only, Openshift supports a list of cidrs.
+- ovs daemon set is expected to be running. Its not part of ovn.
```

Comment:
Maybe "OVS daemons or daemonset are expected to be already running."  I don't think ovn-kube cares whether it's a daemonset or running on the host, as long as the ovsdb socket is available somehow.